web/packages/teleport/src/Discover/Shared/SetupAccess/useUserTraits.ts

```diff
@@ -135,10 +137,13 @@ export function useUserTraits() {
           }
         });
 
-        nextStep({
-          kubeUsers: [...newDynamicKubeUsers],
-          kubeGroups: [...newDynamicKubeGroups],
-        });
+        nextStep(
```

Comment:
if auto discovering, does duplicated users/groups gets de-duplicated in the network call? i'd add a test for this

Reply:
Yep it's deduplicated.